chromadb/db/index/hnswlib.py

```diff
@@ -141,7 +145,9 @@ def add(self, ids, embeddings, update=False):
                 labels.append(next_label)
```

Comment:
Is weird that sometimes we store the UUID vs the id but leaving this for now to minimize the bugs i might introduce.